### DISCUSSION TITLE
(Feature request) Compact Sidebar View

### DISCUSSION THREAD

**Discussion by asyncmeow** [TriliumNext/Trilium#7807] [Q&A]
### Describe feature

I'm working on a custom theme for Trilium, and as part of that I'd like to make a more compact looking sidebar. I tried doing this with the following CSS:
```css
.fancytree-node {
    height: 20px;
}
```
However, it resulted in this when I try to move tree nodes:

https://github.com/zadam/trilium/assets/109551954/2fd92a56-cf07-4dd8-9f51-c27793d7192f

Is there a way to do what I'm trying to do without breaking that? If not, could a CSS variable be added in a future release to control the height of each tree node?

**Comment by SiriusXT** [comment]
I don't see any issues from the video

**Comment by asyncmeow** [comment]
The line showing where the note being moved will be dropped doesn't seem to line up with anything if you change the line height - ie. it sometimes shows the line over other stuff, and the line doesn't really correspond with where it would be dropped (forgot to show it in that video, but it's something else I've noticed) On May 17, 2023 08:47, SnnBcd ***@***.***> wrote:
I don't see any issues from the video

**Comment by zadam** [comment]
Unfortunately I don't know. I assume there will be some problem with CSS elsewhere - I noticed that e.g. the children elements of `.fancytree-node` still have a bigger height. Not sure how fancytree calculates the position of these drop indicators.
